js/components/EmailInstructions.vue

```diff
@@ -0,0 +1,17 @@
+<template>
```

Comment:
So small nitpicking in general. The convention we came up with is having a src folder. Where all the js srces are.
And then we have build it into `js`.

But I could tackle that in a later PR.